File: app/src/release/java/com.meiji.daily/SdkManager.java
package com.meiji.daily;

import android.content.Context;

import okhttp3.OkHttpClient;


/**
 * Created by Meiji on 2017/5/2.
 */

public class SdkManager {
    public static void initStetho(Context context) {
    }

    public static OkHttpClient.Builder initInterceptor(OkHttpClient.Builder builder) {
        return builder;
    }
}

File: app/src/release/java/com.meiji.daily/SdkManager.kt
package com.meiji.daily

import android.content.Context

import okhttp3.OkHttpClient


/**
 * Created by Meiji on 2018/1/19.
 */

object SdkManager {

    fun initStetho(context: Context) {
    }

    fun initInterceptor(builder: OkHttpClient.Builder): OkHttpClient.Builder {
        return builder
    }
}
